amqp/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rabbitmq:3-management
+
+COPY enabled_plugins /etc/rabbitmq/
+COPY rabbitmq.conf /etc/rabbitmq/
+COPY definitions.json /etc/rabbitmq
```

Comment:
Trivial thing, but can you please merge these as `COPY enabled_plugins rabbitmq.conf definitions.json /etc/rabbitmq/` to avoid two unnecessary extra docker layers?

Reply:
More generally, do we really need our own container just for this? Can't we use the official container and add in /etc/rabbitmq/ as a volume? This would also make it easier to change the configuration later on.

Reply:
hmm I created our own Dockerfile because at one point i wanted to change the entrypoint, but scrapped that later on. I'll try it with a simple volume.

Reply:
This might be what we are looking for: https://kubernetes.io/docs/concepts/storage/volumes/#configmap

However, I'm fine with merging the PR with the current cockpit/amqp container (and the other fixes), and keeping this as a future optimization.